cypress/e2e/login_management/studentFirstLogin.feature

```diff
@@ -0,0 +1,37 @@
+@pr
+@stable_test
+Feature: First Login - Login of new student user by SVS
+
+    As a user, I want to login the first time with username and password into SVS
+
+    Scenario Outline: User makes a direct login with username and password
+        Given I am logged in as '<user>' on '<env>'
+        And Created student 'John' 'Student' with email 'john.student@mail.tld'
+        And I am on the students management page
+        And Registering student 'John' with email 'john.student@mail.tld'
+        And Going to student edit page for 'John' with email 'john.student@mail.tld'
+        And Changing the password to 'Password1!'
```

Comment:
Why password change? Is not needed for first login.

Reply:
but is needed to know password to login otherwise should pick up the generated password